src/include/security/helps.rb

```diff
@@ -352,9 +331,48 @@ def initialize_security_helps(include_target)
         "EXTRA_SERVICES"                            => _(
           "<P>Every running service is a potential target of a security attack. Therefore it is recommended to turn off all services which are not used by the system.</P>"
         )
-      } 
+      }
+
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
+
+      # Boot dialog help 2/4, diferent message based on default
+      if ::Security::CtrlAltDelConfig.default == "reboot"
+        help += _(
+          "<p><b>Interpretation of Ctrl + Alt + Del</b>:\n" +
```

Comment:
Use a multiline string here instead of the `+` operator, just replace it by backslash `\`.